udata/api/specs.py

```diff
@@ -0,0 +1,33 @@
+from apispec import APISpec
+from apispec.ext.marshmallow import MarshmallowPlugin
+from apispec_webframeworks.flask import FlaskPlugin
+from flask_apispec.extension import FlaskApiSpec
+
+from udata.core.dataset.apiv2 import get_dataset_search, get_resources_paginated, get_specific_dataset_by_id, get_specific_resource_by_rid
+from udata.core.organization.apiv2 import get_organization_search
+from udata.core.reuse.apiv2 import get_reuse_search
+from udata.core.user.apiv2 import get_users_list, post_new_user
+
+
+def init_app(app):
+    app.config.update({
+        'APISPEC_SPEC': APISpec(
+            title="Udata",
+            version="1.0.0",
+            openapi_version="3.0.2",
+            plugins=[MarshmallowPlugin()],
+        ),
+        'APISPEC_SWAGGER_URL': '/swagger/',
+        'APISPEC_SWAGGER_UI_URL': '/swagger-ui/',
```

Comment:
I think we don't need a swagger ui, if I understand correctly, could we deactivate it?

Reply:
We actually could have one to access it locally or on dev env.
I set it back, but it showed "No API definition provided.". I tried to look into it, but did not succeed. I think we can do without it for now but should keep it in mind.